app/src/main/java/com/codepath/apps/critterfinder/fragments/SwipeablePetsFragment.java

```diff
@@ -129,8 +127,8 @@ public void onAdapterAboutToEmpty(int itemsInAdapter) {
     public void onScroll(float scrollProgressPercent) {
         View view = mCardContainer.getSelectedView();
         view.findViewById(R.id.swipe_card_background).setAlpha(0); // hide the background frame on the card
-        view.findViewById(R.id.item_swipe_right_indicator).setAlpha(scrollProgressPercent < 0 ? -scrollProgressPercent : 0);
-        view.findViewById(R.id.item_swipe_left_indicator).setAlpha(scrollProgressPercent > 0 ? scrollProgressPercent : 0);
+        view.findViewById(R.id.item_swipe_right_indicator).setAlpha(scrollProgressPercent < 0 ? (-1.f * .60f * scrollProgressPercent) : 0);
+        view.findViewById(R.id.item_swipe_left_indicator).setAlpha(scrollProgressPercent > 0 ? (.60f * scrollProgressPercent) : 0);
     }
```

Comment:
this ensures our alpha never hits 1.0 so you can see through the icons even at maximum swipe distance.